crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -46,10 +46,15 @@ impl SystemInformationDiagnosticsPlugin {
 /// [`SystemInformationDiagnosticsPlugin`] for more information.
 #[derive(Debug, Resource)]
 pub struct SystemInfo {
+    /// OS name and version.
     pub os: String,
+    /// System kernel version.
     pub kernel: String,
+    /// CPU brand, for example 'Apple M1 Max'.
```

Comment:
I would prefer not to have an example there, I don't think it adds anything

Reply:
Removed.

sysinfo calls this "brand", but I found that confusing - I would have expected "branch" to just be the manufacturer name.

It looks like "what type of CPU is this" is referred to as "brand string" on Mac, "model name" on Linux and "processor name" on Windows.

"Model name" seemed like the clearest choice.